packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -71,8 +71,11 @@ export const TokensList: React.FC<TokensListProps> = (props) => {
   }
 
   const backgroundColor =
-    (tokens['background-colors'].find((token) => token.name === 'color-background-body')?.value as string) ??
-    'rgb(255,255,255)';
+    (tokens['background-colors'].find((token) => token.name === 'color-background-body')?.value as string) ?? '#fff';
+
+  const inverseBackgroundColor =
+    (tokens['background-colors'].find((token) => token.name === 'color-background-body-inverse')?.value as string) ??
+    '#121C2D';
```

Comment:
chore: can you add some comments above these explaining what they're for? I'm having a hard time grokking this code at a glance so a comment can help future maintainers